n_t_with_s/README.md

```diff
@@ -0,0 +1,158 @@
+# Readme for N(t) with selection example
+
+This directory contains the code to run analysis of
+demographic inference using multiple programs, with 
+identical resulting data from any demographic model 
+found in `stdpopsim` as input to each. 
+This provides an example of how standardized 
+population simulations may be used to compare methods.
+
+The present analyses tests how demographic inference 
+is influenced by selection.
+
+The Snakemake workflow includes the necessary pipeline
+for simulation, analyses, and plotting in an efficient manner.
+Simply choose your parameters in a config file, 
+and let snakemake handle the rest. 
+(For large runs, the use of a cluster is highly encouraged)
+
+_NOTE_: This requires a bleeding edge install of `msprime` at the 
+moment. 
+
+## Workflow
+
+The analysis includes three programs for predicting effective population 
+size through time(`n_t`): 
+[msmc](https://github.com/stschiff/msmc/issues/23),
+[stairwayplot](https://sites.google.com/site/jpopgen/stairway-plot), and
+[smc++](https://github.com/popgenmethods/smcpp).
+There are four target rules that can be executed with the given parameters: 
+`compound_msmc`,
+`compound_smcpp`,
+`compound_stairwayplot`, 
+or you can run all three on the same simulated data with rule `all`.
+
+To run an analysis, create a directory (wherever you want)
+where all results, and intermediate
+files will be stored. Next, create and place a file named `config.json` in it.
+The json file must contain key : value combos described below. An example 
+might look like this:
+
+```json
+{
+    "seed" : 12345,
+    "population_id" : 0,
+    "num_samples_per_population" : [20, 0, 0],
+    "num_sampled_genomes_msmc" : "2,8",
+    "num_msmc_iterations" : 20,
+    "replicates" : 10,
+    "species" : "HomSap",
+    "model" : "OutOfAfrica_3G09",
+    "genetic_map" : "HapmapII_GRCh37",
+    "chrm_list" : "chr22,chrX",
+    "mask_file" : "masks/HapmapII_GRCh37.mask.bed"
```

Comment:
lets set the annotation and the DFE here in the config file. this will make the interface easier, e.g., 
```suggestion
    "mask_file" : "masks/HapmapII_GRCh37.mask.bed"
    "dfe_id": "Gamma_K17"
    "annotation_id": "ensembl_havana_104_exons"
```

Reply:
done